config-optional/cert-manager/use-cert-manager-istio-ingress.yml

```diff
@@ -0,0 +1,25 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+#! This certificate will cause cert-manager to create the secret istio-ingressgateway-certs
+---
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: istio-ingressgateway
+  namespace: istio-system
+spec:
+  secretName: istio-ingressgateway-certs
+  dnsNames:
+  - #@ "*." + data.values.system_domain
+#@ for/end domain in data.values.app_domains:
```

Comment:
for stylistical purposes, lets indent this line and next by 2 spaces, since it applies to array item that follows.